src/main.cc

```diff
@@ -594,14 +594,12 @@ std::vector<Option> options_
        "Specify a backend-specific configuration setting. The format of this "
        "flag is --backend-config=<backend_name>,<setting>=<value>. Where "
        "<backend_name> is the name of the backend, such as 'tensorrt'."},
-  {
-    OPTION_HOST_POLICY, "host-policy", "<string>,<string>=<string>",
-        "Specify a host policy setting associated with a policy name. The "
-        "format of this flag is --host-policy=<policy_name>,<setting>=<value>."
-        "Currently supported settings are 'numa-node', 'cpu-cores'. Note that "
-        "'numa-node' setting will affect pinned memory pool behavior, see "
-        "--pinned-memory-pool for more detail."
-  }
+      {OPTION_HOST_POLICY, "host-policy", "<string>,<string>=<string>",
```

Comment:
Revert this change.

Reply:
I believe this was a clang format but I can revert this change as well?

Reply:
Note the extra comma at the end. That might be impacting format.